spec/javascripts/support/polyfills/String.startsWith.js

```diff
@@ -0,0 +1,5 @@
+if (!String.prototype.startsWith) {
```

Comment:
Is this polyfill required in a Chrome only addon?

Reply:
Not for Chrome, but it is to keep phantomjs (2.1.1) happy when running jasmine in the CI mode